.travis.yml

```diff
@@ -1,24 +1,21 @@
-language: java
-dist: trusty
+language: generic
```

Comment:
Note to self: changes here were borrowed from Louis's branch, might have introduced Java 11 issues?

Reply:
I think we can roll back some of these changes, they were needed because of a travis bug essentially, and they claim to have fixed the bug.  We should be able to specify openjdk8 on bionic now like before.

Reply:
OracleJDK 8 is just gone from bionic, there's no easy way to install it without a bunch of nonesense.  I think we can just drop the oracle jdk build completely.